xm-commons-migration-db/src/main/java/com/icthh/xm/commons/migration/db/config/DatabaseConfiguration.java

```diff
@@ -98,7 +101,8 @@ public LocalContainerEntityManagerFactoryBean entityManagerFactory(
             DataSource dataSource,
             MultiTenantConnectionProvider multiTenantConnectionProviderImpl,
             CurrentTenantIdentifierResolver currentTenantIdentifierResolverImpl,
-            LocalValidatorFactoryBean localValidatorFactoryBean) {
+            LocalValidatorFactoryBean localValidatorFactoryBean,
+            List<EntityScanPackageProvider> entityScanPackageProviderList) {
```

Comment:
will it work when outbox not added to classpath and no EntityScanPackageProvider present?